addons/point_of_sale/static/src/xml/Screens/ProductScreen/Orderline.xml

```diff
@@ -36,8 +36,7 @@
                         </em>
                         <span> </span><t t-esc="props.line.get_unit().name" />
                         at
-                        <t t-if="props.line.display_discount_policy() == 'without_discount' and
-                            env.pos.round_decimals_currency(props.line.get_unit_display_price()) &lt; env.pos.round_decimals_currency(props.line.get_taxed_lst_unit_price())">
+                        <t t-if="props.line.display_discount_policy() == 'without_discount' and env.utils.compareCurrencies(props.line.get_unit_display_price(), props.line.get_taxed_lst_unit_price()) lt 0">
```

Comment:
find a better name or since it's just used once, no need for a util function.

Reply:
I removed the utility function. I kept the logic replacing `round_decimals_currency` with `roundCurrency`.